app/services/idv/steps/threat_metrix_step_helper.rb

```diff
@@ -12,7 +12,7 @@ def threatmetrix_view_variables
       end
 
       def generate_threatmetrix_session_id
-        flow_session[:threatmetrix_session_id] = SecureRandom.uuid if !updating_ssn
+        flow_session[:threatmetrix_session_id] = SecureRandom.uuid if !updating_ssn?
```

Comment:
I recommend including refactors like this (esp. when affecting shared functionality like this) in a separate PR prior to creating the pull request for the feature.

Reply:
Since you are updating this, it may be a good idea to run a manual test with ThreatMetrix enabled on your local stack. Per [Sonia's message](https://gsa-tts.slack.com/archives/C0NGESUN5/p1680303292864869), the configuration involves adding the following to `application.yml`.

```yaml
development:
  proofing_device_profiling: enabled
  lexisnexis_threatmetrix_mock_enabled: true
```

Reply:
I did this and it all worked (tests passed as well).

I did only select `pass`, I didn't try `reject` or any of the other choices.